src/components/ui/radio-group.svelte

```diff
@@ -0,0 +1,89 @@
+<script>
+  import { createEventDispatcher } from 'svelte';
+
+  export let items;
+  export let name;
+  export let classname = null;
+  export let selected = {};
+  export let isColor = false;
+  export let isLabel = false;
+  export let isLabelGreen = false;
+  export let labelPosition = 'right';
+  export let labelClass = null;
+
+  $: selectedId = selected.id + 1 || null;
+
+  let dispatch = createEventDispatcher();
+  const changeRadio = item => {
+    selectedId = item.id + 1;
+    dispatch('change', item);
+  };
+
+  const isWhite = item => {
+    if (!item.color || !isColor) return false;
+    if (item.color.toUpperCase() === '#FFF' || item.color.toUpperCase() === '#FFFFFF') return true;
```

Comment:
If condition didn't match, will return undefined because of no default/fallback `return false;`
```suggestion
    return item.color.toUpperCase() === '#FFF' || item.color.toUpperCase() === '#FFFFFF';
```

Reply:
Good notice